tests/gordo_components/data_provider/test_ncs_reader.py

```diff
@@ -109,3 +109,25 @@ def test_load_series_dry_run(dates, ncs_reader):
         dates[0], dates[1], valid_tag_list_no_asset, dry_run=True
     ):
         assert len(frame) == 0
+
+
+@patch(
+    "gordo_components.data_provider.ncs_reader.NcsReader.ASSET_TO_PATH",
+    {
+        "1776-troc": os.path.join(
+            os.path.dirname(os.path.realpath(__file__)), "data", "datalake"
+        )
+    },
+)
+@pytest.mark.parametrize("remove_bad_values", [True, False])
+def test_load_series_with_filter_bad_data(dates, ncs_reader, remove_bad_values):
+    valid_tag_list = normalize_sensor_tags(["TRC-322"])
+    frame = ncs_reader.load_series(
```

Comment:
This isn't a frame, but rather a series generator, no?

Reply:
This is true, it is a series generator.